scripts/src/main/resources/scripts/command/mvn

```diff
@@ -319,17 +319,17 @@ function doRunBuild() {
     if [ "${?}" == 2 ]
     then
       # Maven version is lower than 4.0 (3.x)
-	  maven_arguments="${MAVEN_ARGS}"
-	fi
+	    maven_arguments="${MAVEN_ARGS}"
+	  fi
```

Comment:
did those changes happen by accident? Seems the indentation was messed by your editor somehow.